addons/sale/wizard/sale_order_cancel_views.xml

```diff
@@ -24,9 +24,9 @@
                     <group col="2">
                         <field name="recipient_ids"
                                widget="many2many_tags_email"
-                               context="{'force_email': True,
-                                         'show_email': True,
-                                         'no_create_edit': True}"/>
+                               options="{'no_quick_create': True}"
```

Comment:
```suggestion
                               options="{'no_create_edit': True}"
```
? 🤔 
Since before it was `no_create_edit: true` in the context. 🤷🏼‍♂️